contracts/tasks.ts

```diff
@@ -5,7 +5,7 @@ const CONTRACT_ALLOW_LIST_ADDRESS = "0x0200000000000000000000000000000000000000"
 const MINT_ADDRESS = "0x0200000000000000000000000000000000000001"
 const TX_ALLOW_LIST_ADDRESS = "0x0200000000000000000000000000000000000002"
 const FEE_MANAGER_ADDRESS = "0x0200000000000000000000000000000000000003"
-const REWARD_MANAGER_ADDDRESS = "0x0200000000000000000000000000000000000004"
+const REWARD_MANAGER_ADDRESS = "0x0200000000000000000000000000000000000004"
```

Comment:
Not to be nit picky or inflate the number of PRs, but if it's a small standalone fix that's separate from the purpose of the PR, it's generally best to do it in its own trivial PR rather than batching it together in a bigger one.

Reply:
I've broken out these changes into a separate PR: https://github.com/ava-labs/subnet-evm/pull/1740